common/writeThoughFile.go

```diff
@@ -44,19 +45,26 @@ var RootShareRegex = regexp.MustCompile(`(^\/\/[^\/]*\/?$)`)
 
 func isRootPath(s string) bool {
 	shortParentDir := strings.ReplaceAll(ToShortPath(s), OS_PATH_SEPARATOR, AZCOPY_PATH_SEPARATOR_STRING)
-	return  RootDriveRegex.MatchString(shortParentDir)  ||
-		RootShareRegex.MatchString(shortParentDir)  ||
+	return RootDriveRegex.MatchString(shortParentDir) ||
+		RootShareRegex.MatchString(shortParentDir) ||
 		strings.EqualFold(shortParentDir, "/")
 }
 
-
 func CreateParentDirectoryIfNotExist(destinationPath string, tracker FolderCreationTracker) error {
```

Comment:
any way to add a UT for this

Reply:
Added a test that mocks the problematic and unproblematic behaviors!